docs/template/public/less/_common.less

```diff
@@ -0,0 +1,356 @@
+// Common styles
+
+@import '_variables';
+
+body.kss-body {
+    background: @white;
+    overflow: visible;
+    font-size: 15px;
+    font-family: 'Open Sans', sans-serif;
+    padding: 0;
+    max-width: 100%;
+    padding-top: 100px;
+    padding-right:0!important;
+}
+
+#kss-wrapper {
+    height: auto;
+}
+
+.active.demo-container {
+    background-color: @screen-bg_gd;
+    padding: 15px;
+}
+
+#kss-header h1 {
+    font-size: 1.5em;
+    margin-top: 5/3em;
+    margin-left: 4/3em;
+}
+
+h1.kss-title {
+    color: @blue-heading;
+    margin-bottom: 30px;
+    font-weight: 500;
+    font-family: 'Dosis', sans-serif;
+}
+
+#kss-main {
+    padding-left: 15px;
+    padding-bottom: 40px;
+
+    &.col-md-12{
+        max-width: 1000px;
+    }
+
+}
+
+.kss-mod-list {
+    width: 100%;
+}
+
+.kss-mod-desc {
+    display: inline-block;
+    width: 69%;
+    padding: 10px 0;
+}
+
+.kss-mod-head {
+    border-top: 1px solid @grey-border;
+    padding: 20px 0 20px;
+    margin-top: 20px;
+}
+
+.kss-mod-name {
+    width: 30%;
+    display: inline-block;
+    text-align: right;
+    padding: 10px;
+    margin: 0;
+    vertical-align: top;
+    font-size: 13px;
+    line-height: 1.42857143;
+    word-break: break-all;
+    word-wrap: break-word;
+    color: @typography_gd;
+    background-color: @kss-grey;
+    border: 1px solid @kss-grey-border;
+    border-radius: 3px;
+    font-family: Courier, "Courier New", monospaced;
+  
+    p {
+        display: block;
+    }
+}
+
+.kss-mod-example {
+    display: block;
+    padding-bottom: 20px;
+    & > td {
+        display: block;
+    }
+}
+
+.kss-markup {
+    margin: 20px 0 40px;
+}
+
+.kss-depth-1 > h1 {
+    font-size: 2.5em * .950;
+}
+
+.kss-depth-2 > h1 {
+    font-size: 2.5em * .900;
+}
+
+.kss-depth-3 > h1 {
+    font-size: 2.5em * .850;
+}
+
+.kss-depth-4 > h1 {
+    font-size: 2.5em * .800;
+}
+
+.kss-depth-5 > h1 {
+    font-size: 2.5em * .750;
+}
+
+.kss-depth-6 > h1 {
+    font-size: 2.5em * .700;
+}
+
+// Styleguide specifc overwrites
+.kss-modifier-example {
+    padding-bottom: 20px;
+    padding-top: 20px;
+
+    .popover {
+        position: relative!important;
+        z-index: 1!important;
+        display: block!important;
+    }
+    > [class^="hyicon-"],
+    > [class*=" hyicon-"] {
+        font-size: 35px;
+    }
+    > .nav-pill {
+        float: none!important;
+    }
+}
+
+.kss-lead{
+    font-weight: bold;
+}
+
+// shared styles across pages 
+#patterns-list-pages,
```

Comment:
the anew of the file is common, but it looks like these are specific to the patterns page